src/creature.cpp

```diff
@@ -1542,3 +1542,23 @@ void Creature::check_dead_state() {
         die( nullptr );
     }
 }
+
+std::pair<std::string, nc_color> const &Creature::get_creature_attitude( Attitude att )
```

Comment:
This name is very misleading. Should be something like `attitude_name` or `attitude_ui_data`.